index.html

```diff
@@ -11053,6 +12346,64 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
+				<p>Authors MUST NOT use <code>aria-brailleroledescription</code> without providing a <em>more specific</em> <code>aria-roledescription</code>. In general, <code>aria-brailleroledescription</code> should only be used in rare cases when a <code>aria-roledescription</code> does not suffice.</p>
+				<p> When using <code>aria-brailleroledescription</code>, authors SHOULD also ensure that:</p>
+				<ol>
+					<li>The element to which <code>aria-brailleroledescription</code> is applied has a valid <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role or has an implicit WAI-ARIA role semantic.</li>
+					<li>The value of <code>aria-brailleroledescription</code> is not empty or does not contain only whitespace characters.</li>
+					<li>The value of <code>aria-brailleroledescription</code> consists of a string of either Unicode with no Unicode Braille Patterns (U+2800..U+28FF) or consists of a string of only Unicode Braille Patterns (U+2800..U+28FF) while not only containing Braille Pattern dots-0 (U+2800).</li>
+				</ol>
+				<p class="note">Note that <a>Assistive Technologies</a> with Braille support can convert <code>aria-roledescription</code> content to Braille. In addition, assistive technologies will be able to customize such Braille output according to user preferences. Using only <code>aria-roledescription</code> is <strong>almost always</strong> the better user experience and authors are <strong>strongly discouraged</strong> from using <code>aria-brailleroledescription</code> to replicate <code>aria-roledescription</code>. Instead, <code>aria-brailleroledescription</code> is meant be used only when <code>aria-roledescription</code> cannot provide an adequate Braille representation, i.e., when a specialized Braille description is very different from a text description converted to Braille. It is very important to note that when using <code>aria-brailleroledescription</code> authors are solely responsible to align the attribute value with the document language and clearly communicate the use of this attribute to the user. This is even more important when the value consists of Unicode Braille Patterns because <a>Assistive technologies</a> will pass such content directly to the user without applying user specific Braille translations; in general, authors are <strong>strongly discouraged</strong> from using Unicode Braille Patterns in <code>aria-brailleroledescription</code>.
```

Comment:
Change:
> authors are solely responsible to align the attribute value 

to:
> authors are solely responsible for aligning the attribute value 

Also, I don't understand this phrase - how can authors do this? Can this phrase just be deleted?
> and clearly communicate the use of this attribute to the user

Reply:
I missed this earlier -- sorry @carmacleod. I'll be happy to make some changes but since this particular note is identical to the one for brailleroledescription, would you be ok with revisiting it after both PRs are merged? That way changes are more easily done for both.

As for the question, this came out of the various discussion for the note. The idea is to try to warn authors that when they use these attributes, they risk causing confusion and they need to mitigate that confusion. I'd compare this to role=application where authors are told to provide instructions without a lot of specifics.

Reply:
I have opened https://github.com/w3c/aria/issues/1205 to address this after aria-braillelabel is merged.

Reply:
Thanks!